Semantic Scholar.js

```diff
@@ -118,12 +118,10 @@ function parseDocument(doc, url) {
 			item.issue = volumeAndIssue[1];
 		}
 		
-		if (rawData.hasPdf) {
-			let paperLink = rawData.links.filter(function (link) {
-				return link.linkType === 's2';
-			})[0].url;
+		if (rawData.hasPdf && (rawData.primaryPaperLink.linkType === 's2'
+			|| rawData.primaryPaperLink.linkType == 'arxiv')) {
```

Comment:
Do we want the PDFs from arXiv also automatically attached here when saving from SemanticScholar? Alternatively we could also just link to them.

Anyway, we can for save the arXiv ID in the extra field as well, cf. https://github.com/zotero/translators/blob/master/arXiv.org.js#L188

Reply:
Thanks for the review! I'd say yes, as I'm not sure there's much functionally different between the hosting locations. I think it fits with semantic scholar's role as an aggregator to grab the papers too.